### DISCUSSION TITLE
Time Control and Employtees

### DISCUSSION THREAD

**Discussion by PVRAMID** [YetiForceCompany/YetiForceCRM#17033] [How to]
I am using Yetiforce as a training software (Not it's primary use case, however it works for us).

I'm using the time control feature as a reporting tool.

However when I create a new time control record, I can't get the record to show up in the employee's Time Control Tab on their details page. It shows up in the time control module on the calander when I set up the "Assigned To", however in relations, I have it linked to an employee but it wont show up in the employees details page. 

How can I fix this?
![TcNtb-Dishonest-0404](https://user-images.githubusercontent.com/75082160/233164251-0d813d53-845a-4227-9b61-5d587ccbaf11.png)
![XL6jq-Thrifty-0404](https://user-images.githubusercontent.com/75082160/233164255-ea2bc427-cdb7-4f0f-983d-75c2241d99bf.png)

**Comment by garryatbrighton** [comment]
"Assigned To" or "Shared with users" should point employee than records will show

**Reply by PVRAMID** [reply]
The employees that I want these records to be related to do not have "user accounts" and they shouldn't have user accounts. Therefore this doesn't work.

**Reply by apcloic** [reply]
@PVRAMID 
Tested on gitstable, I was able to link a time control to an Employee and it appears in Employee related time control tab as expected.

"Assigned to" and "Shared with" are fields related to Users module, not Employee module.
So you have to use the "Company" field, select Employees in the drop-down module list and then choose an employee.

![time_control_employee](https://user-images.githubusercontent.com/2930560/233328365-9eb35eb9-9fcd-4336-a2db-1a61d21cf139.gif)

**Comment by PVRAMID** [comment]
I do not have this same layout

https://user-images.githubusercontent.com/75082160/233470324-e2205b7a-c6df-4823-9854-0821096c3795.mp4

No matter whom the record is assigned to, the time record does not show up in the related employees list. I do not want the "Assigned to" to have any bearing as to whether or not this displays in a users time record list.

**Comment by s2b-git** [comment]
Users module could be linked to employee in the way that when certain (for instance) checkbox is ticked the user becoming employee and subsequent relation could be also set in Employees module

**Reply by apcloic** [reply]
@s2b-git 
Thanks for sharing, I didn't know about this feature.
It would be nice to know which checkbox have to be checked.

**Reply by CHILLTH** [reply]
> @s2b-git Thanks for sharing, I didn't know about this feature. It would be nice to know which checkbox have to be checked.

I believe that was feature improvement or new item suggestiion. By the way, pretty logical
